### ISSUE TITLE
Error: That port is already in use

### ISSUE THREAD

**Issue by nitrocode** [edelight/kitchen#114]
When I run the server command, I receive an error message that says the port is already in use. 

    $ python manage.py runserver

    Validating models...
    
    0 errors found
    August 13, 2015 - 15:45:27
    Django version 1.6.1, using settings 'kitchen.settings'
    Starting development server at http://127.0.0.1:8000/
    Quit the server with CONTROL-C.
    Error: That port is already in use.

I tried to grep for port 8000 to see what was in use and it shows me beam.smp

    $ lsof -Pnl +M -i4 | grep 8000
    beam.smp    1923      999  402u  IPv4   23991      0t0  TCP 127.0.0.1:8000 (LISTEN)

beam.smp seems to be part of RabbitMQ which is packaged by Chef?

How do I get around this port issue?

**Comment by nitrocode** [comment]
First, make sure that your python installation is correctly set to /usr/bin/python and if it's set to /opt/opscode's python, then django won't install properly using pip.

    which python

Found out that django server defaults to port 8000. To get around the issue above, run it on a different port. I had to use the real IP address instead of localhost so replace 1.1.1.1 with your actual IP.

    python manage.py runserver 1.1.1.1:9000

I also went through some dependency hell.

    sudo apt-get install python-dev
    sudo pip install pycrypto
    sudo pip install ecdsa
